CONTRIBUTING.md

```diff
@@ -0,0 +1,101 @@
+# Contributing to InteractiveSideMenu
+
+:+1: First of all, big thanks for your desire to contibute in this project :+1:
+
+## How to contribute
+
+The easiest way to contribute is to *report us about a bug* or *propose a new feature*. Use [GitHub Issues](https://github.com/handsomecode/InteractiveSideMenu/issues) to say us about it.
+Please, feel free to send your suggestions concerning any library issue, documentation and ReadMe typos, [Sample](./Sample) improvements, and any other enhancement. **We really appreciate your participation!**
+
+### How to ask a question
+
+Before asking a question, please, look through [ReadMe](./README.md) and [Sample application](./Sample). Then take a look into [already asked GitHub Issues](https://github.com/handsomecode/InteractiveSideMenu/issues) (even closed). If you still not found an answer, feel free to [open an Issue](https://github.com/handsomecode/InteractiveSideMenu/issues/new) and ask us your question. It'd be great to mark it with prepared `question` label.
+
+### How to report a bug
+
+As it was mentioned above, the standard way to report a bug is [GitHub Issues](https://github.com/handsomecode/InteractiveSideMenu/issues).
+
+Explain the problem and include additional details to help maintainers reproduce the problem:
+
+- **Use a clear and descriptive title** for the issue to identify the problem.
+- **Describe the exact steps which reproduce the problem** in as many details as possible.
+- **Provide specific examples** to demonstrate the steps. Include links to files or GitHub projects, or copy/pasteable snippets, which you use in those examples. If you're providing snippets in the issue, use Markdown code blocks.
+- **Describe the behavior you observed** after following the steps and point out what exactly is the problem with that behavior.
+- **Explain which behavior you expected** to see instead and why.
+- **Include screenshots and animated GIFs** which show you following the described steps and clearly demonstrate the problem.
+- **If you're reporting that library crashed**, include a crash report with a stack trace.
+
+Include details about your configuration and environment:
+
+- **Which version of InteractiveSideMenu** are you using?
+- Do you test on the **Simulator** or a **Real device**?
+- **iOS version** that you test with.
+- **XCode version** if it could be helpful in your case.
+
+### How to request an enhancement
+
+**Enhancements** are features that you might like to suggest to a project, but aren't necessarily bugs/problems with the existing code. There is *a label* for enhancments in [Github's Issues](https://github.com/handsomecode/InteractiveSideMenu/issues), so you can tag issues as `enhancement`, and thereby allow us to prioritize issues/bugs reported to the project.
+
+
+### How to say us about your application
```

Comment:
I agree with it. I think readme more appropriate place because potentially more people will see it because most of the users read readme but not too many of them read contributing file. So, potentially we can get more links to apps.